### PR TITLE
Added handling of large audio files

### PR DESCRIPTION
**Pull request by siddharthakhuntia-lohum**

### CODE REVIEW

File: process_job.py [hunk 1]
```diff
@@ -12,18 +13,22 @@
...
         callback (function): The callback function to handle the result of the task.
-    
+
     """
-    
+
     try:
-        print(f"Job received ${job} in thread ${threading.currentThread().getName()}", job)
+        print(
+            f"Job received ${job} in thread ${threading.currentThread().getName()}\n", job)
```
Comment (rishabh-lohum) [inline process_job.py:25]: Print Looks weird?

File: utils/get_audio.py [hunk 2]
```diff
@@ -31,25 +31,35 @@ def download_youtube_audio(url):
...
+
         filename = f"{key}.mp3"
         output_file = update_download_path(filename)
-        audio_clip.write_audiofile(output_file)
-        print(f"Audio downloaded successfully as '{output_file}'")
+
+        audio_clip.write_audiofile(
+            output_file, bitrate=bitrate)
+
+        print(
```
Comment (rishabh-lohum) [inline utils/get_audio.py:59]: Here also

File: utils/get_audio.py [hunk 3]
```diff
@@ -31,25 +31,35 @@ def download_youtube_audio(url):
...
         video_clip = mp.VideoFileClip(url)
+
         audio_clip = video_clip.audio
+
         filename = f"{key}.mp3"
         output_file = update_download_path(filename)
-        audio_clip.write_audiofile(output_file)
-        print(f"Audio downloaded successfully as '{output_file}'")
+
+        audio_clip.write_audiofile(
```
Comment (rishabh-lohum) [inline utils/get_audio.py:56]: This call also ,formatting is weird?

File: process_job.py (hunk 1 above)
Comment (siddharthakhuntia-lohum) [reply process_job.py:25]: Auto Formatting on save, max characters on a line is generally 80, so this print statements exceed that

**Review by rishabh-lohum — COMMENTED** [review]
Small comments regarding formatting

**Pull request merged**